src/facilities/index.js

```diff
@@ -267,6 +267,12 @@ class Facilities {
       );
     }
 
+    if (!isPlainObject(options)) {
```

Comment:
What if passed undefined? It's usually OK to not pass an options object to a call for most libraries.

Reply:
I agree with this comment.

Reply:
Default parameter is good! Needs less error checking that way.